packages/nuxt/src/core/nitro.ts

```diff
@@ -11,14 +11,20 @@ import escapeRE from 'escape-string-regexp'
 import { defu } from 'defu'
 import fsExtra from 'fs-extra'
 import { dynamicEventHandler } from 'h3'
-import type { Nuxt, RuntimeConfig } from 'nuxt/schema'
+import type { Nuxt, NuxtOptions, RuntimeConfig } from 'nuxt/schema'
 // @ts-expect-error TODO: add legacy type support for subpath imports
 import { template as defaultSpaLoadingTemplate } from '@nuxt/ui-templates/templates/spa-loading-icon.mjs'
 import { version as nuxtVersion } from '../../package.json'
 import { distDir } from '../dirs'
 import { toArray } from '../utils'
 import { ImportProtectionPlugin } from './plugins/import-protection'
 
+export const logLevelMapReverse: Record<NuxtOptions['logLevel'], NitroConfig['logLevel']> = {
```

Comment:
We probably don't need to export this internal map

Reply:
```suggestion
const logLevelMapReverse: Record<NuxtOptions['logLevel'], NitroConfig['logLevel']> = {
```

Reply:
Done~